components/ErrorFallback/ErrorFallback.tsx

```diff
@@ -0,0 +1,24 @@
+export const ErrorFallback = () => {
+  return (
+    <div
+      role="alert"
+      className="min-h-screen rounded bg-red-50 p-4 text-center shadow md:p-8 lg:p-16"
+    >
+      <h1 className="font-serif text-4xl text-indigoGray-90">
+        Something went wrong :(
+      </h1>
+      <p className="text-indigoGray-40">
+        Please try refershing the page or contact us at{' '}
```

Comment:
```suggestion
Please try refreshing the page or contact us at{' '}
```

nitpick:  Little typo here 😉